service/runner_process.go

```diff
@@ -158,7 +158,24 @@ func (p *process) Wait() {
 			if err != nil {
 				p.log.Error().Err(err).Msgf("Wait on %d failed", proc.Pid)
 			} else if ps.ExitCode() != 0 {
-				p.log.Info().Int("exitcode", ps.ExitCode()).Msgf("Wait on %d returned", proc.Pid)
+				if ws, ok := ps.Sys().(syscall.WaitStatus); ok {
+					l := p.log.Info()
+					if ws.Exited() {
+						l = l.Int("exit-status", ws.ExitStatus())
+					}
+
+					if ws.Stopped() {
+						l = l.Str("stop-signal", ws.StopSignal().String())
+					}
+
+					if ws.Signaled() {
+						l = l.Str("signal", ws.Signal().String())
+					}
+
+					l.Int("trap-cause", ws.TrapCause()).Msgf("Wait on %d returned", proc.Pid)
```

Comment:
@maierlars can we add "CoreDump" and "Continued" as well?

    if ws.CoreDump() {
      ...
   }

  if ws.Continued() {
     ...
  }